package.json

```diff
@@ -26,6 +26,10 @@
     "yargs": "^15.0.2"
   },
   "devDependencies": {
-    "eslint": "^6.8.0"
+    "eslint": "^6.8.0",
+    "shelljs": "^0.8.3"
+  },
+  "engines": {
+    "node": ">=10.16.0"
```

Comment:
note: just pushed up a commit bumping this to 12. Since this PR has been open we've shifted to requiring 12